//
// Student License - for use by students to meet course requirements and
// perform academic research at degree granting institutions only.  Not
// for government, commercial, or other organizational use.
//
// abcdefg_NTV_AVP.h
//
// Code generation for function 'abcdefg_NTV_AVP'
//

#ifndef ABCDEFG_NTV_AVP_H
#define ABCDEFG_NTV_AVP_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void abcdefg_NTV_AVP(double P_init, double V_init, double A_init, double P_wayp,
                     double V_wayp, double A_wayp, double A_min, double J_max,
                     double J_min, double T, double t[7]);

#endif
// End of code generation (abcdefg_NTV_AVP.h)
